app/views/users/registrations/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "users/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_field :bio, autofocus: true, autocomplete: "bio" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+  
+  <%= f.hidden_field :post_counter, :value => 0%>
+  <%= f.hidden_field :photo, :value => 
+    'https://www.seekpng.com/png/full/301-3012952_this-free-clipart-png-design-of-blank-avatar.png'%>
+<% end %>
+
+<%= render "users/shared/links" %>
```

Comment:
- [ ] Your sign-up page should match the design. You can see from the design that all the input boxes and buttons are aligned to the center of the page, but yours seems to be left aligned within another container. The input boxes should also have more length and cover more spaces on the page.

  ![image](https://user-images.githubusercontent.com/36542986/225286064-f0de9e54-c3d5-4f87-80f7-4541267fbc57.png)

Reply:
done, thanks!